clu/src/lib.rs

```diff
@@ -188,6 +188,13 @@ fn linux_init(config: Arc<RwLock<settings::RitaSettingsStruct>>) -> Result<(), E
         &network_settings.wg_private_key,
     )?;
 
+    // Yield the mut lock
+    drop(network_settings);
```

Comment:
it's going to get dropped in like 5 lines and there's nothing that may stop it. Are you just trying to be explicit here?

Reply:
hmm typically I try and access/clone rather than doing a drop but I guess it's the same.